library/src/main/java/io/constructor/data/model/search/Option.kt

```diff
@@ -0,0 +1,3 @@
+package io.constructor.data.model.search
+
+data class Option(val count: Int, val value: String?)
```

Comment:
Can we rename to facet option?

Reply:
its under `search` package so I thought its enough, but sure